src/store/reducers/list.js

```diff
@@ -0,0 +1,40 @@
+import { ALL_LIST, ADD_LIST, EDIT_LIST, DELETE_LIST } from "../types";
+const initialState = {
+  lists: [],
+  todosInLists: [],
+};
+
+export default function (state = initialState, actions) {
+  switch (actions.type) {
+    case ALL_LIST:
+      const { lists } = actions.payload;
```

Comment:
there was no need to declare a new variable here and assign it to actions.payload
you could also do it like this
return {
        ...state,
        lists: actions.payload,
      };

Reply:
return {
...state,
lists: actions.payload.lists,
};
thats works !